### DISCUSSION TITLE
hwaccel_args for nvidia GPU and hevc stream?

### DISCUSSION THREAD

**Discussion by tfcarlin** [blakeblackshear/frigate#7366] [General]
Hi,

The following is the output from System->FFPROBE for one of my cameras.

`[{"return_code":0,"stderr":"","stdout":{"programs":[],"streams":[{"avg_frame_rate":"15/1","codec_long_name":"H.265/HEVC(HighEfficiencyVideoCoding)","height":2160,"width":3840},{"avg_frame_rate":"0/0","codec_long_name":"AAC(AdvancedAudioCoding)"}]}},{"return_code":0,"stderr":"","stdout":{"programs":[],"streams":[{"avg_frame_rate":"20/1","codec_long_name":"H.264/AVC/MPEG-4AVC/MPEG-4part10","height":512,"width":896},{"avg_frame_rate":"0/0","codec_long_name":"AAC(AdvancedAudioCoding)"}]}}]`

I would like to use the nvidia GPU to decode the stream.

I am using the following just fine with H.264 cameras.

`hwaccel_args: preset-nvidia-h264`

What is the correct hwaccel_agrs for nvidia GPU and hevc.?

Thanks!

Tim

**Comment by NickM-27** [comment]
`preset-nvidia-h265`

**Comment by tfcarlin** [comment]
PERFECT, thanks!
